app/processing/siti-agri/validate-batch.js

```diff
@@ -27,4 +21,9 @@ const isValidSchema = (batchHeader) => {
   return true
 }
 
-module.exports = validate
+const validateLineTotals = (paymentRequests) => {
+  return paymentRequests
+    .every(a => convertToPence(a.value) === getTotalValueInPence(a.invoiceLines, 'value'))
```

Comment:
any reason for `a` being the temporary variable here?

Reply:
No idea, it was already like that.  I've amended to `x` to be consistent with elsewhere.